mold.h

```diff
@@ -160,20 +160,20 @@ class Warn {
 inline u64 align_to(u64 val, u64 align) {
   if (align == 0)
     return val;
-  assert(__builtin_popcount(align) == 1);
+  assert(std::popcount(align) == 1);
```

Comment:
That'll give a different result if input >= 4G. But the new behavior is better. (Applies to align_down below too.)